Makefile

```diff
@@ -173,8 +173,14 @@ $(STAGING_DIR)/$(CHART_NAME)-$(CHART_VERSION): helm-generate
 	cp -r charts/$(CHART_NAME) $(STAGING_DIR)/$(CHART_NAME)-$(CHART_VERSION)
 
 $(STAGING_DIR)/$(CHART_NAME)-$(CHART_VERSION).tgz: $(STAGING_DIR)/$(CHART_NAME)-$(CHART_VERSION)
+	@# Update Chart.yaml with CHART_VERSION and APP_VERSION
 	sed -r -i.bak -e 's%^version: .*%version: $(CHART_VERSION)%g' $(STAGING_DIR)/$(CHART_NAME)-$(CHART_VERSION)/Chart.yaml
 	sed -r -i.bak -e 's%^appVersion: .*%appVersion: "$(APP_VERSION)"%g' $(STAGING_DIR)/$(CHART_NAME)-$(CHART_VERSION)/Chart.yaml
+
+	@# Update README.md with CHART_VERSION and APP_VERSION
+	sed -r -i.bak -e 's%{{ CHART_VERSION }}%$(CHART_VERSION)%g' $(STAGING_DIR)/$(CHART_NAME)-$(CHART_VERSION)/README.md
```

Comment:
does the `.bak` file need to be cleaned up? We should also probably add .bak to`.gitignore` if it ends up in the current directory.

Reply:
I haven't seen these linger when running the targets locally, so I think we're good.  The main agent using these will be ephemeral GH runners on releases.

Reply:
is there a reason to create backup files at all when originals are in Git? (I personally don't love them in automation - esp given that we repeatedly overwrite the same backup anyway)

Reply:
Yea, at one point I thought some versions of sed required a backup file name.  But that could be outdated knowledge.  I'll test updating it to just `-i ''` and see if GH runners are happy (it works fine on my Mac).

Reply:
Made some updates in https://github.com/spinkube/spin-operator/pull/155/commits/d64faa3d741fa595d2771137f537940d2accc75d.  A bit awkward what with converting multiple CRD GH urls to the one combined yaml url attached to a release... and now I realize for all of the pre-release charts we publish from main, we'll be pointing to non-existent GH releases in this README.

Taking a step back, I wonder if we just replace most of the instructions in this README with a link to https://www.spinkube.dev/docs/spin-operator/installation/installing-with-helm/ and do away with the maintenance burden and duplication of instructions?  WDYT?

Reply:
`sed -i` and `sed -i''` are the GNU sed versions - i guess I can live with backups if we want BSD sed to work 😅